### ISSUE TITLE
Question about package installation

### ISSUE THREAD

**Issue by CRZ37** [OrangeSodahub/SceneCraft#19]
Hi,

Thank you for your outstanding work on this project.

I'm currently setting up the environment and ran into an issue. When I use pip install -e . in the project's root directory to install based on pyproject.toml, I noticed that the scenecraft module itself is not installed.

To fix this, I tried adding include = ["scenecraft*"] to the package discovery section. However, after doing this and reinstalling, I encountered a new problem.

I was wondering if you could point out what I might have done wrong in my steps. Here is the new error I'm seeing:

![Image](https://github.com/user-attachments/assets/0563b132-324b-48e9-b83a-0c73070eafde)

Here is my conda environment：
name: scenecraft
channels:
  - pytorch
  - nvidia
  - nvidia/label/cuda-11.8.0
  - https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/free/
  - conda-forge
  - defaults
dependencies:
  - _libgcc_mutex=0.1
  - _openmp_mutex=4.5
  - alsa-lib=1.2.14
  - aom=3.9.1
  - attr=2.5.1
  - blas=1.0
  - brotli-python=1.1.0
  - bzip2=1.0.8
  - ca-certificates=2025.4.26
  - cairo=1.18.4
  - certifi=2025.4.26
  - cffi=1.17.1
  - charset-normalizer=3.4.2
  - cpython=3.10.18
  - cuda-cccl=11.8.89
  - cuda-command-line-tools=11.8.0
  - cuda-compiler=11.8.0
  - cuda-cudart=11.8.89
  - cuda-cudart-dev=11.8.89
  - cuda-cuobjdump=11.8.86
  - cuda-cupti=11.8.87
  - cuda-cuxxfilt=11.8.86
  - cuda-documentation=11.8.86
  - cuda-driver-dev=11.8.89
  - cuda-gdb=11.8.86
  - cuda-libraries=11.8.0
  - cuda-libraries-dev=11.8.0
  - cuda-memcheck=11.8.86
  - cuda-nsight=11.8.86
  - cuda-nsight-compute=11.8.0
  - cuda-nvcc=11.8.89
  - cuda-nvdisasm=11.8.86
  - cuda-nvml-dev=11.8.86
  - cuda-nvprof=11.8.87
  - cuda-nvprune=11.8.86
  - cuda-nvrtc=11.8.89
  - cuda-nvrtc-dev=11.8.89
  - cuda-nvtx=11.8.86
  - cuda-nvvp=11.8.87
  - cuda-profiler-api=11.8.86
  - cuda-runtime=11.8.0
  - cuda-sanitizer-api=11.8.86
  - cuda-toolkit=11.8.0
  - cuda-tools=11.8.0
  - cuda-version=12.9
  - cuda-visual-tools=11.8.0
  - dav1d=1.2.1
  - dbus=1.16.2
  - ffmpeg=7.1.1
  - filelock=3.18.0
  - font-ttf-dejavu-sans-mono=2.37
  - font-ttf-inconsolata=3.000
  - font-ttf-source-code-pro=2.038
  - font-ttf-ubuntu=0.83
  - fontconfig=2.15.0
  - fonts-conda-ecosystem=1
  - fonts-conda-forge=1
  - freetype=2.13.3
  - fribidi=1.0.10
  - gdk-pixbuf=2.42.12
  - gds-tools=1.4.0.31
  - gettext=0.24.1
  - gettext-tools=0.24.1
  - gmp=6.3.0
  - gmpy2=2.2.1
  - graphite2=1.3.13
  - h2=4.2.0
  - harfbuzz=11.2.1
  - hpack=4.1.0
  - hyperframe=6.1.0
  - icu=75.1
  - idna=3.10
  - intel-openmp=2022.0.1
  - jinja2=3.1.6
  - lame=3.100
  - lcms2=2.17
  - ld_impl_linux-64=2.43
  - lerc=4.0.0
  - level-zero=1.22.4
  - libabseil=20250127.1
  - libasprintf=0.24.1
  - libasprintf-devel=0.24.1
  - libass=0.17.3
  - libblas=3.9.0
  - libcap=2.75
  - libcblas=3.9.0
  - libcublas=11.11.3.6
  - libcublas-dev=11.11.3.6
  - libcufft=10.9.0.58
  - libcufft-dev=10.9.0.58
  - libcufile=1.14.0.30
  - libcufile-dev=1.4.0.31
  - libcurand=10.3.10.19
  - libcurand-dev=10.3.0.86
  - libcusolver=11.4.1.48
  - libcusolver-dev=11.4.1.48
  - libcusparse=11.7.5.86
  - libcusparse-dev=11.7.5.86
  - libdeflate=1.24
  - libdrm=2.4.124
  - libegl=1.7.0
  - libexpat=2.7.0
  - libffi=3.4.6
  - libflac=1.4.3
  - libfreetype=2.13.3
  - libfreetype6=2.13.3
  - libgcc=15.1.0
  - libgcc-ng=15.1.0
  - libgcrypt-lib=1.11.1
  - libgettextpo=0.24.1
  - libgettextpo-devel=0.24.1
  - libgl=1.7.0
  - libglib=2.84.2
  - libglvnd=1.7.0
  - libglx=1.7.0
  - libgomp=15.1.0
  - libgpg-error=1.55
  - libhwloc=2.11.2
  - libiconv=1.18
  - libjpeg-turbo=3.1.0
  - liblapack=3.9.0
  - liblzma=5.8.1
  - libnpp=11.8.0.86
  - libnpp-dev=11.8.0.86
  - libnsl=2.0.1
  - libnvjpeg=11.9.0.86
  - libnvjpeg-dev=11.9.0.86
  - libogg=1.3.5
  - libopenvino=2025.0.0
  - libopenvino-auto-batch-plugin=2025.0.0
  - libopenvino-auto-plugin=2025.0.0
  - libopenvino-hetero-plugin=2025.0.0
  - libopenvino-intel-cpu-plugin=2025.0.0
  - libopenvino-intel-gpu-plugin=2025.0.0
  - libopenvino-intel-npu-plugin=2025.0.0
  - libopenvino-ir-frontend=2025.0.0
  - libopenvino-onnx-frontend=2025.0.0
  - libopenvino-paddle-frontend=2025.0.0
  - libopenvino-pytorch-frontend=2025.0.0
  - libopenvino-tensorflow-frontend=2025.0.0
  - libopenvino-tensorflow-lite-frontend=2025.0.0
  - libopus=1.5.2
  - libpciaccess=0.18
  - libpng=1.6.47
  - libprotobuf=5.29.3
  - librsvg=2.58.4
  - libsndfile=1.2.2
  - libsqlite=3.50.0
  - libstdcxx=15.1.0
  - libstdcxx-ng=15.1.0
  - libsystemd0=257.6
  - libtiff=4.7.0
  - libudev1=257.6
  - libunwind=1.6.2
  - liburing=2.10
  - libusb=1.0.29
  - libuuid=2.38.1
  - libva=2.22.0
  - libvorbis=1.3.7
  - libvpx=1.14.1
  - libwebp-base=1.5.0
  - libxcb=1.17.0
  - libxcrypt=4.4.36
  - libxkbcommon=1.10.0
  - libxml2=2.13.8
  - libzlib=1.3.1
  - llvm-openmp=15.0.7
  - lz4-c=1.10.0
  - markupsafe=3.0.2
  - mkl=2022.1.0
  - mpc=1.3.1
  - mpfr=4.2.1
  - mpg123=1.32.9
  - mpmath=1.3.0
  - ncurses=6.5
  - networkx=3.4.2
  - nsight-compute=2022.3.0.22
  - ocl-icd=2.3.3
  - opencl-headers=2024.10.24
  - openh264=2.6.0
  - openjpeg=2.5.3
  - openssl=3.5.0
  - pango=1.56.3
  - pcre2=10.45
  - pillow=11.2.1
  - pip=25.1.1
  - pixman=0.46.0
  - pthread-stubs=0.4
  - pugixml=1.15
  - pulseaudio-client=17.0
  - pycparser=2.22
  - pysocks=1.7.1
  - python=3.10.16
  - python_abi=3.10
  - pytorch=2.1.2
  - pytorch-cuda=11.8
  - pytorch-mutex=1.0
  - pyyaml=6.0.2
  - readline=8.2
  - requests=2.32.3
  - sdl2=2.32.54
  - sdl3=3.2.16
  - setuptools=80.9.0
  - snappy=1.2.1
  - svt-av1=3.0.2
  - sympy=1.14.0
  - tbb=2022.1.0
  - tk=8.6.13
  - torchaudio=2.1.2
  - torchtriton=2.1.0
  - torchvision=0.16.2
  - typing_extensions=4.14.0
  - urllib3=2.4.0
  - wayland=1.23.1
  - wayland-protocols=1.44
  - wheel=0.45.1
  - x264=1!164.3095
  - x265=3.5
  - xkeyboard-config=2.44
  - xorg-libice=1.1.2
  - xorg-libsm=1.2.6
  - xorg-libx11=1.8.12
  - xorg-libxau=1.0.12
  - xorg-libxcursor=1.2.3
  - xorg-libxdmcp=1.1.5
  - xorg-libxext=1.3.6
  - xorg-libxfixes=6.0.1
  - xorg-libxrender=0.9.12
  - xorg-libxscrnsaver=1.2.4
  - yaml=0.2.5
  - zstandard=0.23.0
  - zstd=1.5.7
  - pip:
      - absl-py==2.3.0
      - accelerate==0.19.0
      - addict==2.4.0
      - aiohappyeyeballs==2.6.1
      - aiohttp==3.12.9
      - aiosignal==1.3.2
      - annotated-types==0.7.0
      - anyio==4.9.0
      - appdirs==1.4.4
      - argon2-cffi==25.1.0
      - argon2-cffi-bindings==21.2.0
      - arrow==1.3.0
      - asttokens==3.0.0
      - async-lru==2.0.5
      - async-timeout==5.0.1
      - attrs==25.3.0
      - av==14.4.0
      - babel==2.17.0
      - beautifulsoup4==4.13.4
      - bidict==0.23.1
      - bleach==6.2.0
      - blinker==1.9.0
      - cachetools==6.0.0
      - click==8.2.1
      - colorlog==6.9.0
      - comet-ml==3.49.10
      - comm==0.2.2
      - configargparse==1.7.1
      - configobj==5.0.9
      - contourpy==1.3.2
      - cryptography==45.0.3
      - cycler==0.12.1
      - dash==3.0.4
      - datasets==3.6.0
      - debugpy==1.8.14
      - decorator==5.2.1
      - defusedxml==0.7.1
      - descartes==1.1.0
      - diffusers==0.23.1
      - dill==0.3.8
      - docstring-parser==0.16
      - dulwich==0.22.8
      - einops==0.8.1
      - embreex==2.17.7.post6
      - everett==3.1.0
      - exceptiongroup==1.3.0
      - executing==2.2.0
      - fastjsonschema==2.21.1
      - fire==0.7.0
      - flask==3.0.3
      - fonttools==4.58.1
      - fpsample==0.3.3
      - fqdn==1.5.1
      - frozenlist==1.6.2
      - fsspec==2025.3.0
      - gdown==5.2.0
      - gitdb==4.0.12
      - gitpython==3.1.44
      - grpcio==1.72.1
      - gsplat==1.4.0
      - h11==0.16.0
      - h5py==3.13.0
      - httpcore==1.0.9
      - httpx==0.28.1
      - huggingface-hub==0.25.2
      - imageio==2.37.0
      - importlib-metadata==8.7.0
      - ipykernel==6.29.5
      - ipython==8.37.0
      - ipywidgets==8.1.7
      - isoduration==20.11.0
      - itsdangerous==2.2.0
      - jaxtyping==0.3.2
      - jedi==0.19.2
      - joblib==1.5.1
      - json5==0.12.0
      - jsonpointer==3.0.0
      - jsonschema==4.24.0
      - jsonschema-specifications==2025.4.1
      - jupyter==1.1.1
      - jupyter-client==8.6.3
      - jupyter-console==6.6.3
      - jupyter-core==5.8.1
      - jupyter-events==0.12.0
      - jupyter-lsp==2.2.5
      - jupyter-server==2.16.0
      - jupyter-server-terminals==0.5.3
      - jupyterlab==4.4.3
      - jupyterlab-pygments==0.3.0
      - jupyterlab-server==2.27.3
      - jupyterlab-widgets==3.0.15
      - kiwisolver==1.4.8
      - lazy-loader==0.4
      - lightning-utilities==0.14.3
      - lxml==5.4.0
      - manifold3d==3.1.1
      - mapbox-earcut==1.0.3
      - markdown==3.8
      - markdown-it-py==3.0.0
      - matplotlib==3.10.3
      - matplotlib-inline==0.1.7
      - mdurl==0.1.2
      - mediapy==1.2.4
      - mistune==3.1.3
      - msgpack==1.1.0
      - msgpack-numpy==0.4.8
      - msgspec==0.19.0
      - multidict==6.4.4
      - multiprocess==0.70.16
      - narwhals==1.41.0
      - nbclient==0.10.2
      - nbconvert==7.16.6
      - nbformat==5.10.4
      - nerfacc==0.5.2
      - nerfstudio==1.1.5
      - nerfstudio-register-new==1.0
      - nest-asyncio==1.6.0
      - ninja==1.11.1.4
      - nodeenv==1.9.1
      - notebook==7.4.3
      - notebook-shim==0.2.4
      - numpy==1.26.4
      - nuscenes-devkit==1.1.9
      - open3d==0.19.0
      - opencv-python==4.11.0.86
      - opencv-python-headless==4.10.0.84
      - overrides==7.7.0
      - packaging==25.0
      - pandas==2.3.0
      - pandocfilters==1.5.1
      - parso==0.8.4
      - pathos==0.3.2
      - pexpect==4.9.0
      - platformdirs==4.3.8
      - plotly==6.1.2
      - plyfile==1.1.2
      - pox==0.3.6
      - ppft==1.7.7
      - prometheus-client==0.22.1
      - prompt-toolkit==3.0.51
      - propcache==0.3.1
      - protobuf==3.20.3
      - psutil==7.0.0
      - ptyprocess==0.7.0
      - pure-eval==0.2.3
      - pyarrow==20.0.0
      - pycocotools==2.0.10
      - pycollada==0.9
      - pydantic==2.11.5
      - pydantic-core==2.33.2
      - pygments==2.19.1
      - pyliblzfse==0.4.1
      - pymeshlab==2023.12.post3
      - pyngrok==7.2.9
      - pyparsing==3.2.3
      - pyquaternion==0.9.9
      - python-box==6.1.0
      - python-dateutil==2.9.0.post0
      - python-engineio==4.12.2
      - python-json-logger==3.3.0
      - python-socketio==5.13.0
      - pytorch-msssim==1.0.0
      - pytz==2025.2
      - pyzmq==26.4.0
      - rawpy==0.25.0
      - referencing==0.36.2
      - regex==2024.11.6
      - requests-toolbelt==1.0.0
      - responses==0.18.0
      - retrying==1.3.4
      - rfc3339-validator==0.1.4
      - rfc3986-validator==0.1.1
      - rich==14.0.0
      - rpds-py==0.25.1
      - rtree==1.4.0
      - safetensors==0.5.3
      - scikit-image==0.25.2
      - scikit-learn==1.6.1
      - scipy==1.15.3
      - semantic-version==2.10.0
      - send2trash==1.8.3
      - sentry-sdk==2.29.1
      - setproctitle==1.3.6
      - shapely==2.1.1
      - shtab==1.7.2
      - simple-websocket==1.1.0
      - simplejson==3.20.1
      - six==1.17.0
      - smmap==5.0.2
      - sniffio==1.3.1
      - soupsieve==2.7
      - splines==0.3.0
      - stack-data==0.6.3
      - svg-path==6.3
      - tensorboard==2.19.0
      - tensorboard-data-server==0.7.2
      - tensorly==0.9.0
      - termcolor==3.1.0
      - terminado==0.18.1
      - threadpoolctl==3.6.0
      - tifffile==2025.5.10
      - timm==0.6.7
      - tinycss2==1.4.0
      - tinycudann==1.7
      - tokenizers==0.13.3
      - tomli==2.2.1
      - torch-fidelity==0.3.0
      - torchmetrics==1.7.2
      - tornado==6.5.1
      - tqdm==4.67.1
      - traitlets==5.14.3
      - transformers==4.29.2
      - trimesh==4.6.11
      - typeguard==4.4.3
      - types-python-dateutil==2.9.0.20250516
      - typing-inspection==0.4.1
      - tyro==0.9.24
      - tzdata==2025.2
      - uri-template==1.3.0
      - vhacdx==0.0.8.post2
      - viser==0.2.7
      - wadler-lindig==0.1.6
      - wandb==0.20.1
      - wcwidth==0.2.13
      - webcolors==24.11.1
      - webencodings==0.5.1
      - websocket-client==1.8.0
      - websockets==15.0.1
      - werkzeug==3.0.6
      - widgetsnbextension==4.0.14
      - wrapt==1.17.2
      - wsproto==1.2.0
      - wurlitzer==3.1.1
      - xatlas==0.0.10
      - xxhash==3.5.0
      - yarl==1.20.0
      - yourdfpy==0.0.57
      - zipp==3.22.0
prefix: /home/crz/miniconda3/envs/scenecraft

Thank you for your time

**Comment by OrangeSodahub** [comment]
@CRZ37 Hi, sorry for the inconvenience. This is due to I delete the 'big2' method before, you can directly comment out (or delete) the line 17, where the 'big2' is defined.
